substrate/frame/balances/src/lib.rs

```diff
@@ -505,11 +511,19 @@ pub mod pallet {
 	#[pallet::genesis_config]
 	pub struct GenesisConfig<T: Config<I>, I: 'static = ()> {
 		pub balances: Vec<(T::AccountId, T::Balance)>,
+
+		#[cfg(feature = "runtime-benchmarks")]
+		pub dev_accounts: (u32, T::Balance, Option<String>),
 	}
 
 	impl<T: Config<I>, I: 'static> Default for GenesisConfig<T, I> {
 		fn default() -> Self {
-			Self { balances: Default::default() }
+			Self {
+				balances: Default::default(),
+
+				#[cfg(feature = "runtime-benchmarks")]
```

Comment:
As the benchmarks are a little bit different to the runtime benchmarks, I would just expose `dev_accounts` always. If not needed, people don't need to set it.

Reply:
> As the benchmarks are a little bit different to the runtime benchmarks,

Could you explain a bit more @bkchr?

Reply:
maybe a link in the code differentiating benchmarks from runtime benchmarks? familiar with just the feature flag `runtime-benchmarks`.